txpipe/twopoint_fourier.py

```diff
@@ -66,9 +66,10 @@ class TXTwoPointFourier(PipelineStage):
         "deproject_syst_clustering": False,
         "systmaps_clustering_dir": '',
         "ell_min": 100,
-        "ell_max": 1500,
-        "n_ell": 20,
-        "ell_spacing": 'log'
+        "ell_max": 3000,
```

Comment:
Can leave these changes in as they don't hurt.

Reply:
Well, if we change nside to 512 again we can't reach that higher ell. What should we use? Should I leave nside 2048 or return to ell_max 1500?

Reply:
yeah, good point!  Can we leave it on the original defaults just because they're faster for the laptop tests?